libmamba/include/mamba/util/flat_set.hpp

```diff
@@ -30,6 +30,7 @@ namespace mamba::util
     public:
 
         using Base = std::vector<Key, Allocator>;
+        using Self = flat_set<Key, Compare, Allocator>;
```

Comment:
bitpicking: I think we can simply use `flat_set` in the declaration and implementation, the template parameters will be deduced.